### PR TITLE
Fix strings with s uppercase

### PR DESCRIPTION
**Pull request by lucasnobredev**
## Summary

This PR is from https://github.com/dotnet/docs/issues/8700.

Fixes dotnet/docs#8700

### CODE REVIEW

File: snippets/csharp/programming-guide/discards/discard-pattern2.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ private static void ProvidesFormatInfo(object obj)
          Console.Write("A null object reference: ");
          Console.WriteLine("Its use could result in a NullReferenceException");
       }
-      else if (obj is var _)
+      else
```
Comment (BillWagner) [inline snippets/csharp/programming-guide/discards/discard-pattern2.cs]: Let's leave this sample as is for the moment.  Issue dotnet/docs#8727 points to the need for a  better sample. I'd rather leave this in place until we make a new sample that shows the usage better.

Comment (rpetrusha) [inline snippets/csharp/programming-guide/discards/discard-pattern2.cs]: Although this change is acceptable, it requires a change to example description. The example is used in [Discards - C# Guide](https://docs.microsoft.com/en-us/dotnet/csharp/discards), where the sentence "It also used the discard pattern to handle non-null objects of any other type." needs to be removed. Could you open a PR to do that?

**Comment by rpetrusha** [comment]
@BillWagner and I reviewed this at about the same time, @lucasnobredev. I agree with him, though, that the final example should be left in place until we come up with a better example. Since you've also made the same update in #425, the last change can be reverted.

**Comment by lucasnobredev** [comment]
@rpetrusha @BillWagner I was reading the article again and it seems like if we remove condition of code and remove the phrase "It also used the discard pattern to handle non-null objects of any other type", the section lost the sense to exist. I'll take this comment to https://github.com/dotnet/docs/issues/8727 and I'll wait.

**Comment by BillWagner** [comment]
I looked at the latest updates, and I'm ready to :shipit:

Let me know what you think @rpetrusha

**Comment by rpetrusha** [comment]
I agree that this is ready to merge, @BillWagner. Thanks, @lucasnobredev. I've tagged this as verify build before merge, since the merge to live build is still running. It can be merged once that build completes and the merge-to-live PR is closed.

**Pull request merged**